### ISSUE TITLE
Package Wmail as an RPM

### ISSUE THREAD

**Issue by longtomjr** [Thomas101/wmail#485]
Pretty much the title. I think WMail would benefit from this as a lot of people are put off when installing non packaged files. It would also make it easier to install on RPM distros.

I am relatively rusty with packaging but I am willing to take on the task. I found it relatively trivial to install via source, but a package would make it a lot easier.

**Comment by 1ShotSniper** [comment]
I would be willing to package this as .pkg.tar.gz for pacman and submit it to the Arch User Repository

**Comment by longtomjr** [comment]
OK. That would be good. I might be able to get this into the fedora repos or the RPM fusion repositories, but otherwise just put it on copr. 

Another thing we need to talk about is maintaining the packages. Also, i think opening a seperate issue for the arch package would be better than posting here @1ShotSniper .

**Comment by longtomjr** [comment]
OK. After two days of work on this, I could not find a way to do a proper build and install. I also could not find documentation on creating RPMs With binaries. 

This turned into a rabbit hole I was not prepared for and unfortunately my knowledge about RPMs is quite limited.

**Comment by 1ShotSniper** [comment]
@longtomjr 

Found something to help out: https://fedoraproject.org/wiki/How_to_create_an_RPM_package

Also, if you can't package the binaries, can't you instead package the source from the 2.0.0 branch?

**Comment by longtomjr** [comment]
Yep, the problem is not packaging something like that, it is packaging node apps. 
Yes packaging from the source is the best way to go, but I have no idea where to start with node and npm. It looks a bit far over my head to do as someone who packaged a basic deb or two and some basic rpms (that have a makefile etc.). Also, finding support on this issue is hard.

**Comment by pauldotknopf** [comment]
This solution could be a good reference.

https://github.com/MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-

They have rpm support.

**Comment by funilrys** [comment]
@1ShotSniper @longtomjr  you can find wmail in AUR ==> https://aur.archlinux.org/packages/wmail-bin/
